test/test_type_promotion.py

```diff
@@ -808,6 +808,80 @@ def test_integer_addcdiv_deprecated(self, device, dtype):
         with self.assertRaisesRegex(RuntimeError, '^Integer division.+is no longer supported+'):
             t.addcdiv_(t, t)
 
+    def test_addcdiv_promotion(self, device):
+        arg1 = torch.rand([5, 5], device=device, dtype=torch.float64)
+        arg2 = torch.rand([5, 5], device=device, dtype=torch.float64)
+        arg3 = torch.rand([1, 5], device=device, dtype=torch.complex128)
+
+        res1 = torch.addcdiv(arg1, arg2, arg3)
+        res2 = arg1 + arg2 / arg3
+
+        self.assertEqual(res1.dtype, res2.dtype)
+        self.assertEqual(res1, res2)
+
+    def test_addcmul_promotion(self, device):
+        arg1 = torch.rand([5, 5], device=device, dtype=torch.float64)
+        arg2 = torch.rand([5, 5], device=device, dtype=torch.float64)
+        arg3 = torch.rand([1, 5], device=device, dtype=torch.complex128)
+
+        res1 = torch.addcmul(arg1, arg2, arg3)
+        res2 = arg1 + arg2 * arg3
+
+        self.assertEqual(res1.dtype, res2.dtype)
+        self.assertEqual(res1, res2)
+
+    def test_complex32_promotion(self, device):
```

Comment:
cc @kshitij12345  -- is it worth testing complex32 type promotion at this point? Further, this kinda seems like an odd PR to add it to.

Reply:
I think it would be nice to have this test since it will start failing once this will be supported. (Should it be added in this PR? I don't think so)

Thanks!

Reply:
i just removed this from the pr